src/bundle/Resources/public/scss/_field-group.scss

```diff
@@ -9,11 +9,47 @@
             margin-bottom: 0;
             padding-bottom: .5rem;
             font-size: 1.125rem;
-            font-weight: 400;  
+            font-weight: 400;
+
+            a {
+                color: $ez-color-secondary;
+
+                &:hover {
+                    text-decoration: none;
+                }
+
+                .ez-icon--collapsed {
+                    display: none;
+                }
+
+                .ez-icon--expanded {
+                    display: inline-block;
+                }
+            }
+
+            a.collapsed {
+                .ez-icon--collapsed {
+                    display: inline-block;
+                }
+
+                .ez-icon--expanded {
+                    display: none;
+                }
+            }
+
+            .ez-icon {
+                fill: $ez-color-secondary;
```

Comment:
You can use `ez-icon--secondary`

Reply:
Fixed https://github.com/ezsystems/ezplatform-admin-ui/pull/860/commits/74642645add985253a5b9050c2405818fc255b20